<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QWidget" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #202020;</string>
  </property>
  <widget class="QLabel" name="monthTerm">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>20</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: white;
font: 14pt &quot;Sans Serif Collection&quot;;
}</string>
   </property>
   <property name="text">
    <string>Month Term</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="differentiated">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>100</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
padding-left: 10px;
background-color: #323232;
color: #ffffff;
font: 14pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}

QRadioButton:hover {
background-color: #3B3B3B;
}

QRadioButton:pressed {
background-color: #282828;
color: #cecece;
}

QRadioButton::indicator {
width: 14px;
height: 14px;
border-radius: 9px;
border: 2px solid  #ffffff;
}

QRadioButton::indicator:checked {
width: 10px;
height: 10px;
border: 4px solid  #9690c2;
border-radius: 9px;
}</string>
   </property>
   <property name="text">
    <string>Differentiated</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCalculate">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #b5adeb;
color: #2e2c3c;
font: 14pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}

QPushButton:hover {
background-color: #9690c2;
}

QPushButton:pressed {
background-color: #9690c2;
color: #605c7c;
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="annuity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
padding-left: 10px;
background-color: #323232;
color: #ffffff;
font: 14pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}

QRadioButton:hover {
background-color: #3B3B3B;
}

QRadioButton:pressed {
background-color: #282828;
color: #cecece;
}

QRadioButton::indicator {
width: 14px;
height: 14px;
border-radius: 9px;
border: 2px solid  #ffffff;
}

QRadioButton::indicator:checked {
width: 10px;
height: 10px;
border: 4px solid  #9690c2;
border-radius: 9px;
}</string>
   </property>
   <property name="text">
    <string>Annuity</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="monthTermValue">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>50</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
background-color: #323232;
color: #ffffff;
font: 14pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}

QSpinBox:hover {
background-color: #3B3B3B;
}

QSpinBox::up-button { width: 0px;
border:0;
}

QSpinBox::down-button { width: 0px;
border: 0;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <number>600</number>
   </property>
  </widget>
  <widget class="QLabel" name="interestRate">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>20</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: white;
font: 14pt &quot;Sans Serif Collection&quot;;
}</string>
   </property>
   <property name="text">
    <string>Interest rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="creditAmount">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>20</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: white;
font: 14pt &quot;Sans Serif Collection&quot;;
}</string>
   </property>
   <property name="text">
    <string>Credit amount</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="interestRateValue">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
background-color: #323232;
color: #ffffff;
font: 14pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}

QDoubleSpinBox:hover {
background-color: #3B3B3B;
}

QDoubleSpinBox::up-button { width: 0px;
border:0;
}

QDoubleSpinBox::down-button { width: 0px;
border: 0;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <double>365.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="creditAmountValue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
background-color: #323232;
color: #ffffff;
font: 14pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}

QDoubleSpinBox:hover {
background-color: #3B3B3B;
}

QDoubleSpinBox::up-button { width: 0px;
border:0;
}

QDoubleSpinBox::down-button { width: 0px;
border: 0;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <double>9999999999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="overpayment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
padding-left: 10px;
color: white;
font: 18pt &quot;Sans Serif Collection&quot;;
}</string>
   </property>
   <property name="text">
    <string>Overpayment</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalPayment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
padding-left: 10px;
color: white;
font: 18pt &quot;Sans Serif Collection&quot;;
}</string>
   </property>
   <property name="text">
    <string>Total Payment</string>
   </property>
  </widget>
  <widget class="QLabel" name="monthlyPayment">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
padding-left: 10px;
color: white;
font: 18pt &quot;Sans Serif Collection&quot;;
}</string>
   </property>
   <property name="text">
    <string>Monthly Payments</string>
   </property>
  </widget>
  <widget class="QLabel" name="overpaymentValue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
padding-left: 10px;
background-color: #323232;
color: #ffffff;
font: 18pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalPaymentValue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
padding-left: 10px;
background-color: #323232;
color: #ffffff;
font: 18pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="monthlyPaymentValue">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>220</y>
     <width>261</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser {
padding-left: 10px;
background-color: #323232;
color: #ffffff;
font: 18pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif Collection'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
